plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -575,6 +576,18 @@
     }
   }
 
+  let supportStatus: SupportStatus | undefined = undefined
+  function handleSupportStatusChanged (status: SupportStatus) {
+    supportStatus = status
+  }
+
+  const supportClient = getResource(support.function.GetSupport).then((res) =>
+    res((status) => handleSupportStatusChanged(status))
+  )
+  onDestroy(async () => {
+    await supportClient?.then((support) => support.destroy())
```

Comment:
Why not simply form?
```suggestion
    (await supportClient?)?.destroy()
```

Reply:
Fixed